lib/turn.rb

```diff
@@ -0,0 +1,34 @@
+class Turn
+  GAME_RULES = {
+    rock: {rock: :draw, paper: :lose, scissors: :win},
+    paper: {rock: :win, paper: :draw, scissors: :lose},
+    scissors: {rock: :lose, paper: :win, scissors: :draw}
+  }
+  attr_reader :name, :player_option, :computer_option
+
+  def initialize(options)
```

Comment:
This might be a bit clearer if you pass the specific parts of the session hash in individually, at the moment we're passing a bunch of stuff the class doesn't need along with it.